databuilder/databuilder/extractor/dashboard/apache_superset/__init__.py

```diff
@@ -0,0 +1,175 @@
+import abc
+from functools import reduce
+from json import dumps
+from typing import (
+    Any, Dict, Generator, Iterator, List, Tuple,
+)
+
+import requests
+from bs4 import BeautifulSoup
+from dateutil import parser
+from pyhocon import ConfigFactory, ConfigTree
+
+from databuilder.extractor.base_extractor import Extractor
+
+type_fields_mapping = List[Tuple[str, str, Any, Any]]
+
+
+class ApacheSupersetBaseExtractor(Extractor):
```

Comment:
which superset version does it support? Also I assume it shouldn't apply specific to atlas store only?

Reply:
- the extractor is written to be compliant with databuilder models, second step will be making those models atlas compliant
- I am writing this for Apache Superset 1.1 and will update docs once it's promoted from WIP
- One outstanding thing is chart and query extractor. However, superset is not best aligned with databuilder model - databuilder has DashboardMetadata <> Query <> Chart data model and relationships where in Superset it's more focused on DashboardMetadata <> Chart and it's nearly impossible to get proper query relationships.